### PR TITLE
Movie-listing-liza-minican

### PR DESCRIPTION
**Pull request by liza-minican**
Link to my deployed project: https://movie-listing-project-liza.netlify.app

My reflections on how this week's project turned out:
The project was fun and I liked the potential of Router, buy i am still confused  about many things.  

Things I'd like to have clarified or explained in more detail:
I need to practice using useEffect and useState much more also useParams.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
-    </div>
+    <BrowserRouter>
+    <Switch>
+      <Route path="/"exact>
+        <MovieList />
+      </Route>
+      <Route path="/movies/:id" exact>
+        <MovieDetails />
+      </Route>
+      <Route path="/404">
```
Comment (amandaflod) [inline code/src/App.js:18]: Great that you added the not-found page! :D

File: code/src/components.css/movieDetails.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  color: #fff;
+}
+.vote-red {
+  margin-left: 10px;
+  color: red;
+  font-weight: 400;
+  font-size: 20px;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
```
Comment (amandaflod) [inline code/src/components.css/movieDetails.css:37]: Amazing with some responsiveness to this page ✨

File: code/src/pages/MovieDetails.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      })
+  }
+
+  useEffect (()=> {
+    getMovieDetails(id)
+  },[id]);  
+  console.log(movieDetails);
+
+  return(
+    <article className="movie-details">
```
Comment (amandaflod) [inline code/src/pages/MovieDetails.js:29]: Here you could add the return button as shown in the example by Technigo. I did like this (but I*m sure there are other ways as well 😀)

`<Link className="homepage-link" to="/">
        Back Button
        <p>
          Movies
        </p>
      </Link>`

File: code/src/pages/NotFound.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (amandaflod) [inline code/src/pages/NotFound.js:1]: Again, great job on this! 🚀

**Review by amandaflod — COMMENTED** [review]
Sorry for the delayed review, I completely forgot about it this week! 😅 Amazing code Liza, there was barely anything to comment on! 👏 The only thing I would suggest (if you have the time) is to add the return button to the details page, but other than that, it looks prefect! 🐯😍

File: code/src/pages/MovieList.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    .then((response) => response.json())
+    .then (json => setMovies(json.results));
+},[]);
+
+  return (
+    <section className="movies-list">
+      {movies.map((movie) => (
+        <MovieCard key={movie.id} 
+                  {...movie}/>
+    ))}</section>
```
Comment (EmOlsz) [inline code/src/pages/MovieList.js:24]: Wrong indentations

File: code/src/App.js [hunk 6]
```diff
@@ -1,9 +1,27 @@
...
+        <NotFound />
+      </Route>
+      <Redirect 
+      to="/404"
+      />
+    </Switch>
+    </BrowserRouter>
```
Comment (EmOlsz) [inline code/src/App.js:25]: Wrong indentations

**Pull request closed without merging**